Content.Shared/_RMC14/Xenonids/Parasite/SharedXenoParasiteSystem.cs

```diff
@@ -476,8 +479,9 @@ public override void Update(float frameTime)
                         var message = Loc.GetString("rmc-xeno-infection-insanepain-" + random);
                         _popup.PopupEntity(message, uid, uid, PopupType.LargeCaution);
 
-                        var knockdownTime = infected.BaseKnockdownTime * 10;
-                        InfectionShakes(uid, infected, knockdownTime, knockdownTime, false);
+                        var knockdownTime = infected.BaseKnockdownTime * 2;
+                        var jitterTime = infected.JitterTime * 0;
```

Comment:
Are these changes intended? @Vermidia wrote the original ones, but they are hard to check in 13

Reply:
Yes, supposed to mimic the jitter starting up violently when the larva is bursting